samples/data-grid/basic/data-grid-2-benchmark/demo.js

```diff
@@ -27,7 +27,11 @@ oldDatagridTimer.innerHTML =
 const startNewTime = new Date().getTime();
 var t2 = performance.now();
 const newgrid = new DataGrid.DataGrid2('container', {
-    dataTable: dataTable
+    dataTable: dataTable,
+    rowOptions: {
```

Comment:
Could this potentially, and perhaps unintentionally by user, be overwritten by custom CSS? Would that break things? I consider this styling, and I guess all other styling will be done by CSS?

Possible to get the actual height of cells onload, and use that value?

Reply:
I removed these options and changed them to fully CSS. 😄